sql/updates/world/3.3.5/2017_06_05_99_world_335.sql

```diff
@@ -0,0 +1,47 @@
+DELETE FROM `command` WHERE `name`='reload quest_greeting';
+INSERT INTO `command` (`name`, `permission`, `help`) VALUES
+('reload quest_greeting', 843, 'Syntax: .reload quest_greeting\nReload quest_greeting table.');
+
+DELETE FROM `command` WHERE `name`='reload quest_greeting';
```

Comment:
I hope this is a copy-paste mistake, because it deletes the previous insert. Maybe you meant
```sql
DELETE FROM `command` WHERE `name`='reload quest_greeting_locale';
```